features/dd-sdk-android-trace/build.gradle.kts

```diff
@@ -50,8 +50,14 @@ android {
     }
 }
 
+repositories{
+    mavenLocal()
+}
+
 dependencies {
     api(project(":dd-sdk-android-core"))
+    api("io.opentelemetry:opentelemetry-api:1.4.0")
```

Comment:
why `api` and not `implementation`?

Reply:
me playing with things ...like I mentioned below in the real implementation will be `compile` and clients will add the dependency on their end.